build.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project name="build-rally-connector" basedir="." default="all">
+
+    <property name="BUNDLE_NAME" value="Connector-RALLY"/>
+    <property name="CONNECTOR_DIR" value="rally-impl"/>
```

Comment:
We should rename all "rally" to "agile central" (or agilecentral". 

Rally was bought by CA (a competitor), and renamed "CA Agile Central". Let's not mention CA in the name though :)